app/src/main/java/org/mozilla/fenix/theme/FirefoxColors.kt

```diff
@@ -0,0 +1,140 @@
+    val LightGrey70 = Color(0xFF9F9FAD)
+    val LightGrey80 = Color(0xFF8F8F9D)
+    val LightGrey90 = Color(0xFF80808E)
+
+    // Dark grey should primarily be used for the Dark theme and secondary buttons.
+    val DarkGrey05 = Color(0xFF5B5B66)
+    val DarkGrey10 = Color(0xFF52525E)
+    val DarkGrey20 = Color(0xFF4A4A55)
+    val DarkGrey30 = Color(0xFF42414D)
+    val DarkGrey40 = Color(0xFF3A3944)
+    val DarkGrey50 = Color(0xFF32313C)
+    val DarkGrey60 = Color(0xFF2B2A33)
+    val DarkGrey70 = Color(0xFF23222B)
+    val DarkGrey80 = Color(0xFF1C1B22)
+    val DarkGrey90 = Color(0xFF15141A)
+
+    // Blue is one of our primary accent colors. Used for primary buttons and text links.
+    val Blue05 = Color(0xFFAAF2FF)
+    val Blue10 = Color(0xFF80EBFF)
+    val Blue20 = Color(0xFF00DDFF)
+    val Blue30 = Color(0xFF00B3F4)
+    val Blue40 = Color(0xFF0090ED)
+    val Blue50 = Color(0xFF0060DF)
+    val Blue60 = Color(0xFF0250BB)
+    val Blue70 = Color(0xFF054096)
+    val Blue80 = Color(0xFF073072)
+    val Blue90 = Color(0xFF09204D)
+
+    // Green is primarily used for positive / success / update status in user interface.
+    val Green05 = Color(0xFFE3FFF3)
+    val Green10 = Color(0xFFD1FEEE)
+    val Green20 = Color(0xFFB3FFE3)
+    val Green30 = Color(0xFF87FFD1)
+    val Green40 = Color(0xFF54FFBD)
+    val Green50 = Color(0xFF3FE1B0)
+    val Green60 = Color(0xFF2AC3A2)
+    val Green70 = Color(0xFF008787)
+    val Green80 = Color(0xFF005E5E)
+    val Green90 = Color(0xFF08403F)
+
+    // Yellow is primarily used for attention / caution / warining status in user interface.
+    val Yellow05 = Color(0xFFFFFFCC)
+    val Yellow10 = Color(0xFFFFFF98)
+    val Yellow20 = Color(0xFFFFEA80)
+    val Yellow30 = Color(0xFFFFD567)
+    val Yellow40 = Color(0xFFFFBD4F)
+    val Yellow50 = Color(0xFFFFA436)
+    val Yellow60 = Color(0xFFE27F2E)
+    val Yellow70 = Color(0xFFC45A27)
+    val Yellow80 = Color(0xFFA7341F)
+    val Yellow90 = Color(0xFF960E18)
+
+    // Firefox Orange is only used for branding. Do not use it otherwise!
+    val Orange05 = Color(0xFFFFF4DE)
+    val Orange10 = Color(0xFFFFD5B2)
+    val Orange20 = Color(0xFFFFB587)
+    val Orange30 = Color(0xFFFFA266)
+    val Orange40 = Color(0xFFFF8A50)
+    val Orange50 = Color(0xFFFF7139)
+    val Orange60 = Color(0xFFE25920)
+    val Orange70 = Color(0xFFCC3D00)
+    val Orange80 = Color(0xFF9E280B)
+    val Orange90 = Color(0xFF7C1504)
+
+    // Red is primarily used for attention / error status or a destructive action in user interface.
+    val Red05 = Color(0xFFFFDFE7)
+    val Red10 = Color(0xFFFFBDC5)
+    val Red20 = Color(0xFFFF9AA2)
+    val Red30 = Color(0xFFFF848B)
+    val Red40 = Color(0xFFFF6A75)
+    val Red50 = Color(0xFFFF4F5E)
+    val Red60 = Color(0xFFE22850)
+    val Red70 = Color(0xFFC50042)
+    val Red80 = Color(0xFF810220)
+    val Red90 = Color(0xFF440306)
+
+    // Violet is primarily used for ????? in user interface.
```

Comment:
Reminder to update this documentation. Detail currently missing in Figma. 
@topotropic any wording suggestion here?